include/nuttx/can/can.h

```diff
@@ -246,6 +257,46 @@
  *                   is returned with the errno variable set to indicate the
  *                   nature of the error.
  *   Dependencies:   None
+ *
+ * CANIOC_SET_STATE
+ *   Description:    Set specfic can controller state
+ *
+ *   Argument:       A pointer to an int type that describes the CAN
```

Comment:
It is better to avoid passing pointers in ioctl calls. We can just pass integer here.

Reply:
I replicated the code comments authored by CANIOC_SET_CONNMODES. The code comments of CANIOC_SET_CONNMODES employ pointers.